### PR TITLE
feat: add a Sync Now button to Plaid Settings [CU-b6e72a]

### PR DESCRIPTION
**Pull request by Alchez**
plaid-sync-now

### CODE REVIEW

**Comment by vjFaLk** [comment]
Task linked: [CU-b6e72a Symo - Add a "Sync Now" to Plaid settings.](https://app.clickup.com/t/b6e72a)

File: erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js [hunk 1]
```diff
@@ -15,6 +15,10 @@ frappe.ui.form.on('Plaid Settings', {
 			frm.add_custom_button('Link a new bank account', () => {
 				new erpnext.integrations.plaidLink(frm);
 			});
+
+			frm.add_custom_button(__("Sync Now"), () => {
+				frm.call("sync_now");
```
Comment (vjFaLk) [inline erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js]: Add a freeze

Bonus UX Task - Add a little popup (you know the one that shows on the bottom right) showing how many transactions were pulled with a button to view them.

File: erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.py [hunk 2]
```diff
@@ -19,6 +19,10 @@ def get_link_token():
 		plaid = PlaidConnector()
 		return plaid.get_link_token()
 
+	@staticmethod
```
Comment (vjFaLk) [inline erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.py]: Why `@staticmethod`?

**Review by vjFaLk — CHANGES_REQUESTED** [review]

File: erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.py (hunk 2 above)
Comment (Alchez) [reply erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.py]: It's just to be able to quickly sync up without needing an instance of Plaid Settings. I could also do a direct whitelist call to `enqueue_sync`

File: erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js (hunk 1 above)
Comment (Alchez) [reply erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js]: If it was a direct call, I could control the output, but the call is enqueued and the response isn't passed back here 😅

Comment (vjFaLk) [reply erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.js]: Then a message saying that "It's been enqueued, please check [link to transactions list], yada yada" would work.

**Review by vjFaLk — CHANGES_REQUESTED** [review]
Left comments

**Comment by Alchez** [comment]
@vjFaLk, done

**Comment by vjFaLk** [comment]
@Alchez One last thing...

PRs to upstream? :sweat_smile:

**Comment by Alchez** [comment]
Made upstream to [develop](https://github.com/frappe/erpnext/pull/23602).

**Pull request merged**